locales/bg.json

```diff
@@ -0,0 +1,38 @@
+{
+  "Page": "Страница",
+  "of": "от",
+  "Published with": "Публикувано чрез",
+  "Subscribe": "Абонирай се",
+  "Subscribe to": "Абонирай се",
+  "Subscribed!": "Абонира се!",
+  "Subscribe to our newsletter": "Абонирай се за нашия нюзлетър",
+  "You've successfully subscribed to": "Успешно се абонира за",
+  "Your email address": "Your email address",
```

Comment:
Can you please add this translation? I think is the only one missing ;)

Reply:
Yeah. Done. (: